lib/modules/datasource/hermit/readme.md

```diff
@@ -0,0 +1,22 @@
+By default [Hermit](https://cashapp.github.io/hermit/) looks up packages from the open source project [https://github.com/cashapp/hermit-packages](https://github.com/cashapp/hermit-packages).
+
+Hermit supports [private packages](https://cashapp.github.io/hermit/packaging/private/).
+To get Renovate to find your private packages, follow these steps:
+
+1. perform `hermit search --json` with your private Hermit distribution and save the file to `index.json`
+1. make a GitHub release in your private packages repository named `index` with the asset `index.json` generated in step 1.
+1. setup a CI pipeline to repeat step 1 & 2 on new commits to the private packages repository.
```

Comment:
> I guess you need to follow these steps in your package's repository? It looks like you create a index.json file, and then publish it in a GitHub release?

Yes, it will be needed to be done in package's repository. like [cashapp/hermit-packages](https://github.com/cashapp/hermit-packages).

GitHub action is one way to do the publish. That is easily achievable in many other CI pipelines. That's why I write down the steps but not really outline it in a specific platform.